src/shape_pen.rs

```diff
@@ -139,6 +139,9 @@ fn bez_to_shape(path: &BezPath) -> Shape {
             value: Value::Fixed(vertices),
             ..Default::default()
         },
+        // 1.0 = Clockwise = positive area
+        // 3.0 = Counter-Clockwise = negative area
```

Comment:
We should make a trait and an enum so we can say things like  some_bez.direction() == Direction::Clockwise

Reply:
'll fix that while fixing the conflicts.